### DISCUSSION TITLE
Re-order recipes in a book?

### DISCUSSION THREAD

**Discussion by VL-Renegade** [TandoorRecipes/recipes#3294] [General]
I might have missed something obvious, but i cant seem to re-order recipes in a cookbook.
I use a filter based cookbook, so recipes get added based on a specific label.
It works if I add the label to the recipes in the order i want them in (specifically a certain order, because this is a physical book being digitalised). But sometimes, this does not work.
Have i missed some setting that lets me re-order them?

Besides that, i wanted to make sure the creators and contributors are aware their work is greatly appreciated.
Unfortunately i have no coding skills otherwise i would love to help out.
But i try to help by re-doing the full Dutch translation to have it make more sense using more context specific text instead of literal translations.

**Comment by vabene1111** [comment]
hi, you are sadly correct that there is no way of ordering recipes in a book. This is something that needs to be done in the future after vue 3 is stable at some point. 

for the translation: you are correct that many translated strings lack context. Sadly without any coding skills its probably hard to find the context in any other way than looking at the application. The only thing you could probably do is take the "key" of the translation and search trough the repository on where its used. Then try to understand from the file name and the text snippets around it what it does or where it is in the application.
 
![image](https://github.com/user-attachments/assets/8880c4c3-93b7-4710-9830-01d217e875d4)

**Reply by VL-Renegade** [reply]
Thanks for the tip, i have in fact already completed a lot of the translation doing it exactly like that. 😁
Every time i come across something that needs adjusting, i do it immediately.
Since is use the develop branche, most of the time i can see my adjustments the next day.
